src/Uno.UI/UI/Xaml/Application.Android.cs

```diff
@@ -32,6 +34,19 @@ public Application()
 		{
 			Suspending?.Invoke(this, new ApplicationModel.SuspendingEventArgs(new ApplicationModel.SuspendingOperation(DateTime.Now.AddSeconds(30))));
 		}
+
+		private ApplicationTheme GetDefaultSystemTheme()
+		{		
+			if ((int)Build.VERSION.SdkInt >= 28)
+			{
+				var uiModeFlags = Android.App.Application.Context.Resources.Configuration.UiMode & UiMode.NightMask;
+				if (uiModeFlags == UiMode.NightNo)
+				{
+					return ApplicationTheme.Light;
+				}				
+			}
+			return ApplicationTheme.Dark;
```

Comment:
The default should be `Light`, if the OS level doesn't support dark mode.

Reply:
Even though the UWP default is `Dark`?

Reply:
The reason why I chose `Dark` as the default is that UWP has dark theme resources in the `Default` theme dictionary, not light, but probably makes sense to match the target OS instead

Reply:
Done :-)